ros_buildfarm/workspace.py

```diff
@@ -41,6 +42,32 @@ def clean_workspace(workspace_root):
     if os.path.exists(test_results_dir):
         shutil.rmtree(test_results_dir)
 
+def call_abi_checker(workspace_root, rosdistro_name, env):
+    # TODO: pkgs detection, code based on create_devel_task_generator.py
+    condition_context = {}
+    condition_context['ROS_DISTRO'] = rosdistro_name
```

Comment:
The dict needs to be populate with more information: `ROS_VERSION`, `ROS_PYTHON_VERSION`.

Reply:
Thanks for the hint, 1d6db93